apps/rehabstod/src/components/SettingsDialogContent/SettingsDialogContent.tsx

```diff
@@ -16,7 +17,17 @@ export function SettingsDialogContent({ preferences, onClose }: { preferences: U
     return null
   }
 
-  const isValueBetweenLimits = (max: number, min: number, value: string) => value && Number(value) <= max && Number(value) >= min
+  const isMaxAntalDagarMellanIntygValid = isValueBetweenLimits(
```

Comment:
This has introduced a bug where the user can save empty inputs

Reply:
Good catch, fixed